COPYING

```diff
@@ -1,372 +1,18 @@
-This software includes code from minizip, which is part of zlib;
-in particular, the following files:
-src/contrib/*.[ch]
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
```

Comment:
So what do you think, can we keep the "or later" part?

Reply:
Sure, "or later" is fine with me.